kube_apiserver_metrics/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+***Added***:
+
+* Add etcd.db.total_size for Kubernetes 1.26, 1.27 and 1.28 ([#15940](https://github.com/DataDog/integrations-core/pull/15940))
```

Comment:
```suggestion
* Add etcd.db.total_size for Kubernetes 1.26, 1.27, and 1.28 ([#15940](https://github.com/DataDog/integrations-core/pull/15940))
```

Reply:
Sure, I've updated the PR with the following:

```
diff --git a/kube_apiserver_metrics/CHANGELOG.md b/kube_apiserver_metrics/CHANGELOG.md
index f5486aee91..adbc379efb 100644
--- a/kube_apiserver_metrics/CHANGELOG.md
+++ b/kube_apiserver_metrics/CHANGELOG.md
@@ -4,7 +4,7 @@

 ***Added***:

-* Add etcd.db.total_size for Kubernetes 1.26, 1.27 and 1.28 ([#15940](https://github.com/DataDog/integrations-core/pull/15940))
+* Add etcd.db.total_size for Kubernetes 1.26, 1.27, and 1.28 ([#15940](https://github.com/DataDog/integrations-core/pull/15940))

 ## 4.0.0 / 2023-08-10
```